### ISSUE TITLE
Improve the schedule popover

### ISSUE THREAD

**Issue by jeremywight** [hackgvl/trolley-tracker-web#4]
@Linktheoriginal - @markmcdaniels is going to take a stab at improving the schedule overlay on the trolley-tracker-web. Just wanted to log this issue and introduce you all!

**Comment by bikeoid** [comment]
Welcome Mark - let me know if there are any questions!   Currently we publish the web site as a subfolder to the API, but it is possible to set up a direct FTP also if that makes it easier.

**Comment by MarkMcDaniels** [comment]
Thank you. I'll look at the code over the next few days, and if I have any
questions I'll be in touch.

**Comment by MarkMcDaniels** [comment]
I can't seem to get the site to load locally for some reason. Instead I've
been going through the code to while looking at the live version. I wanted
to change the html to make it easier for myself, but I'm having a hard time
finding where the <span> for the route code is being generated. I'm a
neophyte with django, so it could be something really obvious that I just
don't understand.

Thanks,
Mark

**Comment by bikeoid** [comment]
We have moved the web project from the django server to the ASP.NET server to eliminate one server from the mix.  The current configuration is:

http://trackthetrolley.com has an iframe with the content hosted at http://api.yeahthattrolley.com/ClientWeb , content controlled by https://github.com/codeforgreenville/trolley-tracker-api-dot-net/tree/master/TrolleyTracker/Content/ClientWeb .   Inside that page, the variables
  routedata
  scheduledata
  fulltrolleydata

  are JSON objects populated dynamically on load by the ASP.NET page.  That page contains a div for info-question and info-schedule which read the data objects and create the schedule display.

  js/main.js contains the logic to display and update the data.   I did not get any map display here when I saved the page locally; I only got the route schedule.  There was no error either, so I'm not sure where it goes wrong.

 Also, the dev server at http://yeahthattrolley.azurewebsites.net/ClientWeb will have some active routes if you wish to work with those.
